modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -12,44 +12,29 @@
 # governing permissions and limitations under the License.
 
 from modin.engines.base.io.text.text_file_dispatcher import TextFileDispatcher
-from modin.data_management.utils import compute_chunksize
-from pandas.io.parsers import _validate_usecols_arg
 import pandas
-import csv
-import sys
+from csv import QUOTE_NONE
 
 from modin.config import NPartitions
 
 
 class CSVDispatcher(TextFileDispatcher):
     @classmethod
     def _read(cls, filepath_or_buffer, **kwargs):
+        # Fallback to pandas if parameter is not supported in Modin yet
```

Comment:
It's better to move check parameters support in separate function - for example: `_to_csv_check_support` in https://github.com/modin-project/modin/pull/2715

Reply:
Ok, function was added.